### PR TITLE
Fix keyboard things

### PR DESCRIPTION
**Pull request by bakerboy448**
title

(untested)

### CODE REVIEW

File: pkg/snapshot/synology.go [hunk 1]
```diff
@@ -67,7 +67,7 @@ func GetSynology() (*Synology, error) { //nolint:cyclop
 			syn.Build = strings.Trim(lsplit[1], "\n\"")
 		case "manager":
 			syn.Manager = strings.Trim(lsplit[1], "\n\"")
-		case "vender":
```
Comment (bakerboy448) [inline pkg/snapshot/synology.go:70]: typo or is synology's payload misspelled?

File: pkg/client/handlers.go [hunk 2]
```diff
@@ -125,7 +125,7 @@ func (c *Client) notFound(response http.ResponseWriter, request *http.Request) {
 
 	response.WriteHeader(http.StatusNotFound)
 
-	if err := c.templat.ExecuteTemplate(response, "404.html", nil); err != nil {
+	if err := c.template.ExecuteTemplate(response, "404.html", nil); err != nil {
```
Comment (davidnewhall) [inline pkg/client/handlers.go:128]: you had best put this one.

File: pkg/client/start.go [hunk 3]
```diff
@@ -45,7 +45,7 @@ type Client struct {
 	clientinfo *clientinfo.Config
 	triggers   *triggers.Actions
 	cookies    *securecookie.SecureCookie
-	templat    *template.Template
+	template    *template.Template
```
Comment (davidnewhall) [inline pkg/client/start.go:48]: one extra space here failed the build.

File: pkg/snapshot/synology.go (hunk 1 above)
Comment (davidnewhall) [reply pkg/snapshot/synology.go:70]: you better ask nit what's in his syno file.

**Review by davidnewhall — CHANGES_REQUESTED** [review]

File: pkg/client/handlers.go (hunk 2 above)
Comment (bakerboy448) [reply pkg/client/handlers.go:128]: ?

**Pull request merged**